corehq/apps/translations/templates/download_translations.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'hqwebapp/two_column.html' %}
```

Comment:
Realize this matches other templates in the same directory, but why doesn't it inherit from `base_section.html`? Most pages should inherit from base_section rather than two_column (the former gives you breadcrumbs).